volatility3/framework/plugins/windows/cachedump.py

```diff
@@ -22,7 +22,7 @@ class Cachedump(interfaces.plugins.PluginInterface):
     """Dumps lsa secrets from memory"""
 
     _required_framework_version = (2, 0, 0)
-    _version = (1, 0, 1)
+    _version = (1, 0, 2)
```

Comment:
Sorry, meant this comment for this change...

I mean... you can but there's no need.  Dependencies changing shouldn't require a bump of the plugin version (because the plugin won't necessarily know it externals have changed), but as I say, not a problem, better to bump then all than none of them...  5:)